Cython/Utility/Builtins.c

```diff
@@ -303,6 +303,21 @@ static PyObject *__Pyx_PyLong_AbsNeg(PyObject *n) {
 #define __Pyx_PyNumber_Power2(a, b) PyNumber_Power(a, b, Py_None)
 
 
+//////////////////// divmod_int.proto //////////////////
+
+static CYTHON_INLINE PyObject* __Pyx_divmod_int(int a, int b){
+    PyObject* result_tuple;
+    PyObject* args;
+    div_t res = div(a, b);
+    args = PyTuple_Pack(2, res.quot, res.rem);
+    if (unlikely(!args))
+        return NULL;
+    result_tuple = PyObject_Call((PyObject*)&PyTuple_Type, args, NULL);
```

Comment:
I don't understand this line. Why not just return `args`?

Reply:
Replaced by `Py_BuildValue` at the second commit.